Functions/InputOutput/xASL_io_DcmtkRead.m

```diff
@@ -185,49 +197,65 @@
 
 % Read the nested fields
 if isfield(header,'SharedFunctionalGroupsSequence')
-	timingItem = header.SharedFunctionalGroupsSequence.Item_1.MRTimingAndRelatedParametersSequence.Item_1;
+	if iscell(header.SharedFunctionalGroupsSequence)
+		timingItem = header.SharedFunctionalGroupsSequence{1}.MRTimingAndRelatedParametersSequence{1};
+	else
+		timingItem = header.SharedFunctionalGroupsSequence.Item_1.MRTimingAndRelatedParametersSequence.Item_1;
+	end
 	if isfield(timingItem,'RepetitionTime')
 		header.RepetitionTime = timingItem.RepetitionTime;
 	end
 end
 
 %% Philips fields
-if isfield(header,'PerFrameFunctionalGroupsSequence')
-	echoItem = header.PerFrameFunctionalGroupsSequence.Item_1.MREchoSequence.Item_1;
+if isfield(header, 'PerFrameFunctionalGroupsSequence')
+	privatItem = [];
```

Comment:
e missing privateItem

privat is German ;)

Reply:
Fixed